windows/wsl_distro_install_uninstall/wsl_convert_binary_to_txt.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get the binary file from guest os to local and convert it to text format.
+# Parameters:
+#   source_path: the source file path on remote host
+#   file_name: source file name
+# Return:
+#   new_file: the text format file on local host
+
+- name: "Get file from guest OS to local machine"
+  include_tasks: ../utils/win_get_file.yml
+  vars:
+    win_get_file_src_path: "{{ source_path }}"
+    win_get_file_dst_path: "{{ current_test_log_folder }}"
+
+- name: "Set the fact of file"
+  ansible.builtin.set_fact:
+    src_file: "{{ fetch_file.dest }}"
+    wsl_new_file: "{{ current_test_log_folder }}/{{ file_name }}"
+
+- name: "Covert the binary file to text file"
+  ansible.builtin.shell: "tr -d '\\0' < {{ src_file }} > {{ wsl_new_file }} "
```

Comment:
Do you mean "dos2unix" ?

Reply:
No. dos2unix doesnot work. 
(ansible2.13) root@ubuntu-ansible:~/test/localhost/C:/tmp# dos2unix wsl_install.txt new.txt
dos2unix: Binary symbol 0x0000 found at line 1
dos2unix: Skipping binary file wsl_install.txt
dos2unix: new.txt: No such file or directory
dos2unix: Skipping new.txt, not a regular file.

Reply:
Got it. Thanks~